### PR TITLE
Solving issue #3026

### PR DESCRIPTION
**Pull request by j0rt3g4**
We are solving the issue since MS has moved to the recommended of 64 bits, and updated the links.
Also enhanced the "Workaround" words all around the article.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: mdop/uev-v2/microsoft-user-experience-virtualization--ue-v--20-release-notesuevv2.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ UE-V will roam the Outlook 2010 signature files between devices. However, the de
 
 ### UE-V does not support roaming settings between 32-bit and 64-bit versions of Microsoft Office
 
-We recommend that you install the 64-bit version of Microsoft Office for modern computers. To determine which version you you need, [click here](https://support.office.com/en-us/article/choose-between-the-64-bit-or-32-bit-version-of-office-2dee7807-8f95-4d0c-b5fe-6c6f49b8d261?ui=en-US&rs=en-US&ad=US#32or64Bit=Newer_Versions).
+We recommend that you install the 64-bit version of Microsoft Office for modern computers. To determine which version you you need, [click here](https://support.office.com/article/choose-between-the-64-bit-or-32-bit-version-of-office-2dee7807-8f95-4d0c-b5fe-6c6f49b8d261?ui=en-US&rs=en-US&ad=US#32or64Bit=Newer_Versions).
```
Comment (JohanFreelancer9) [inline mdop/uev-v2/microsoft-user-experience-virtualization--ue-v--20-release-notesuevv2.md]: We recommend that you install the 64-bit version of Microsoft Office for modern computers. To determine which version **you you** need, [click here](https://support.office.com/article/choose-between-the-64-bit-or-32-bit-version-of-office-2dee7807-8f95-4d0c-b5fe-6c6f49b8d261?ui=en-US&rs=en-US&ad=US#32or64Bit=Newer_Versions).

**Remove double word (you)**

File: mdop/uev-v2/microsoft-user-experience-virtualization--ue-v--20-release-notesuevv2.md [hunk 2]
```diff
@@ -32,55 +32,55 @@ This section contains release notes for User Experience Virtualization.
...
 
-WORKAROUND: None.
+**WORKAROUND:** None.
 
 ### UE-V does not support roaming settings between 32-bit and 64-bit versions of Microsoft Office
 
-We recommend that you install the 32-bit version of Microsoft Office for both 32-bit and 64-bit operating systems. To choose the Microsoft Office version that you need, click here. ([http://office.microsoft.com/word-help/choose-the-32-bit-or-64-bit-version-of-microsoft-office-HA010369476.aspx](https://go.microsoft.com/fwlink/?LinkID=247623)). UE-V supports roaming settings between identical architecture versions of Office. For example, 32-bit Office settings will roam between all 32-bit Office instances. UE-V does not support roaming settings between 32-bit and 64-bit versions of Office.
+We recommend that you install the 64-bit version of Microsoft Office for modern computers. To determine which version you you need, [click here](https://support.office.com/en-us/article/choose-between-the-64-bit-or-32-bit-version-of-office-2dee7807-8f95-4d0c-b5fe-6c6f49b8d261?ui=en-US&rs=en-US&ad=US#32or64Bit=Newer_Versions).
```
Comment (JohanFreelancer9) [inline mdop/uev-v2/microsoft-user-experience-virtualization--ue-v--20-release-notesuevv2.md]: We recommend that you install the 64-bit version of Microsoft Office for modern computers. To determine which version **you you** need, [click here](https://support.office.com/article/choose-between-the-64-bit-or-32-bit-version-of-office-2dee7807-8f95-4d0c-b5fe-6c6f49b8d261?ui=en-US&rs=en-US&ad=US#32or64Bit=Newer_Versions).

Please remove double word (you).

**Comment by mypil** [comment]
@j0rt3g4 - please see the comments from JohanFreelancer9, there's a typo error found. You may have to create a new PR if you cannot make any changes to this PR since the content is already merged. Thanks you.

**Comment by j0rt3g4** [comment]
@mypil  It's done the New PR is # 3072. Thank you and I'm sorry for the typo.

**Pull request merged**